tendrl/commons/flows/utils.py

```diff
@@ -47,6 +49,49 @@ def ceph_create_ssh_setup_jobs(parameters):
 
 def gluster_create_ssh_setup_jobs(parameters):
     node_list = parameters['Node[]']
+
+    # Install python-gdeploy on the node
```

Comment:
This function deals with setting up ssh jobs, please move this gdeploy setup in a new function